rust/auction/program/src/instruction.rs

```diff
@@ -30,10 +31,20 @@ pub enum AuctionInstruction {
     /// Create a new auction account bound to a resource, initially in a pending state.
     ///   0. `[signer]` The account creating the auction, which is authorised to make changes.
     ///   1. `[writable]` Uninitialized auction account.
-    ///   2. `[]` Rent sysvar
-    ///   3. `[]` System account
+    ///   2. `[writable]` Auction extended data account.
```

Comment:
Is this a mistake that this is still here?

Reply:
the account is used in the instruction itself. i think the comment looks correct